selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/core/CheSeleniumSuiteModule.java

```diff
@@ -127,25 +126,23 @@ public void configure() {
       install(new CheSeleniumSingleUserModule());
     }
 
-    configureInfrastructureRelatedDependencies();
-    configureTestExecutionModeRelatedDependencies();
-  }
-
-  private void configureInfrastructureRelatedDependencies() {
-    String cheInfrastructure = System.getenv(CHE_INFRASTRUCTURE_VARIABLE);
-    if (cheInfrastructure == null || cheInfrastructure.isEmpty()) {
-      throw new RuntimeException(
-          format(
-              "Che infrastructure should be defined by environment variable '%s'.",
-              CHE_INFRASTRUCTURE_VARIABLE));
-    } else if (cheInfrastructure.equalsIgnoreCase(DOCKER_INFRASTRUCTURE)) {
-      install(new CheSeleniumDockerModule());
-    } else if (cheInfrastructure.equalsIgnoreCase(OPENSHIFT_INFRASTRUCTURE)) {
-      install(new CheSeleniumOpenshiftModule());
-    } else {
-      throw new RuntimeException(
-          format("Infrastructure '%s' hasn't been supported by tests.", cheInfrastructure));
+    final Infrastructure cheInfrastructure =
```

Comment:
Please, restore `configureInfrastructureRelatedDependencies()` method and update its content according to your PR.